aws/sdk/integration-tests/s3/tests/express.rs

```diff
@@ -37,3 +37,64 @@ async fn list_objects_v2() {
         .await
         .unwrap();
 }
+
+#[tokio::test]
+async fn list_objects_v2_in_both_express_and_regular_buckets() {
+    let _logs = capture_test_logs();
+
+    let http_client = ReplayingClient::from_file(
+        "tests/data/express/list_objects_v2_in_both_express_and_regular_buckets.json",
+    )
+    .unwrap();
+    let config = aws_config::from_env()
+        .http_client(http_client.clone())
+        .no_credentials()
+        .region("us-west-2")
+        .load()
+        .await;
+    let config = Config::from(&config)
+        .to_builder()
+        .with_test_defaults()
+        .build();
```

Comment:
this loads credentials from the evironment—either use `.test_credentials` on `aws_config` or skip it and load `SdkConfig` or `s3::Config` directly

Reply:
I think the `no_credentials()` call is preventing it from loading them from environment.

Reply:
> the no_credentials() call is preventing it from loading them from environment.

I think it is.